online-grocery-store/src/components/layout/MainLayout.js

```diff
@@ -8,7 +9,20 @@ const Header = () => {
       <Grid.Col span={2} />
       <Grid.Col span={8}>
         <Flex h="100%" justify="space-between" align="center">
-          <Link href="/">Header</Link>
+          <Link href="/">TEACHER'S SHOP</Link>
+          <Link href="/">
+            <Image radius="md" src="Logo.jpg" h={50} w="auto" />
```

Comment:
Place Teacher's Shop here. Should wrap The logo and the name inside 1 <Link> only